db/migrate/20210114010220_change_tasks_priority_type.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeTasksPriorityType < ActiveRecord::Migration[6.0]
+  def change
+    change_table :tasks do |t|
+      t.change(:priority, :string)
```

Comment:
WARN: if you want to preserve your old data, a migration script would need to be added to this script to convert existing record's priority from integers to strings.